src/components/elements/ButtonTooltipContainer.tsx

```diff
@@ -9,7 +9,7 @@ const ButtonTooltipContainer = ({
 
   return (
     <Tooltip title={title} placement='top' arrow {...props}>
-      <span>{children}</span>
```

Comment:
Yes @martha the reason for adding this was because Tooltips won't appear on `disabled` Button elements without it. There are some places where we use a tooltip to explain the reason why a button is disabled (I think on household composition mgmt page, but maybe others).

Reply:
I see, thanks. I'll revert the change and add accessible text directly to the icon buttons. I will also add a note to https://github.com/open-path/Green-River/issues/7089 if we want to revisit this